server/logging/logging.go

```diff
@@ -75,6 +84,15 @@ func New(config config.KolideConfig, logger log.Logger) (*OsqueryLogger, error)
 		if err != nil {
 			return nil, errors.Wrap(err, "create firehose result logger")
 		}
+	case "pubsub":
+		status, err = NewPubSubLogWriter(
```

Comment:
This needs to be `result, err` because right now it is only setting the status logger.